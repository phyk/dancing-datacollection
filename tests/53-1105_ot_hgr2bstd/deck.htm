<!DOCTYPE html PUBLIC "-//WAPFORUM//DTD XHTML Mobile 1.2//EN" "http://www.openmobilealliance.org/tech/DTD/xhtml-mobile12.dtd">
<HTML>
<HEAD>
  <META http-equiv="Content-Type" content="text/html; charset=utf-8">
  <META name="Author" content="Hessischer Tanzsportverband e.V.">
  <META name="GENERATOR" content="TopTurnierDigital">
  <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
  <meta http-equiv="Pragma" content="no-cache" />
  <meta http-equiv="Expires" content="0" />
  <TITLE>11.05.2024 OT, Hgr.II B Standard &#34;2&#34;</TITLE>
  <link rel="stylesheet" type="text/css" href="topturnier.css">
  <meta name="viewport" content="width=device-width, initial-scale=1" />
</HEAD>
<body><div class="mainback">
<div class="eventhead">
<table border=0 width=100%>
<tr><td>11.05.2024 - OT, Hgr.II B Standard &#34;2&#34;</td><td width=30>
<a class="backbtn" href="index.htm" target="_top">&equiv;</a>
</td></tr>
</table>
</div>
<div class="maincontainer">
<div class="comphead">Deckblatt</div>
<hr class="line">
<div class="extract">
   <TABLE class="tab1">
    <TR>
     <TD class="td2" nowrap width=15%>Veranstalter:</TD>
     <TD class="td5" style="word-break:break-all;word-wrap:break-word;">Hessischer Tanzsportverband</TD>
    </TR>
    <TR>
     <TD class="td2" nowrap>Ausrichter:</TD>
     <TD class="td5" style="word-break:break-all;word-wrap:break-word;">Hessischer Tanzsportverband</TD>
    </TR>
    <TR>
     <TD class="td2" nowrap>Turnierleiter:</TD>
     <TD class="td5">
     <span class="col1">Jungbluth, Kai&nbsp;</span> <span>Tanz-Sport-Club Fischbach&nbsp;</span>
     </TD>
    </TR>
    <TR>
     <TD class="td2" nowrap>Beisitzer:</TD>
     <TD class="td5">
     <span class="col1">Rahaus, Markus&nbsp;</span> <span>Schwarz-Rot-Club Wetzlar&nbsp;</span>
     </TD>
    </TR>
    <TR>
     <TD class="td2" nowrap>Protokoll:</TD>
     <TD class="td5">
     <span class="col1">tanzt, EDV-Team Hessen&nbsp;</span> <span>&nbsp;</span>
     </TD>
    </TR>
    <TR>
     <TD class="td2" nowrap>Wertungsrichter:</TD>
     <TD class="td5"></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>BI:</TD>
     <TD class="td5"> <span class="col1">Fleischer, Georg&nbsp;</span> <span>Gr&uuml;n-Gold-Casino Wuppertal&nbsp;</span></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>CP:</TD>
     <TD class="td5"> <span class="col1">Peinke-Dean, Lutz&nbsp;</span> <span>Tanzsportklub Residenz Dresden&nbsp;</span></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>DK:</TD>
     <TD class="td5"> <span class="col1">Wenzel, Harald&nbsp;</span> <span>Rot-Weiss-Klub Kassel&nbsp;</span></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>DL:</TD>
     <TD class="td5"> <span class="col1">Wied, Dr. Andrea&nbsp;</span> <span>Markgr&auml;fler TSC, M&uuml;llheim&nbsp;</span></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>DR:</TD>
     <TD class="td5"> <span class="col1">Zuber, Dr. Pascal&nbsp;</span> <span>TSC Metropol Hofheim&nbsp;</span></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>EL:</TD>
     <TD class="td5"> <span class="col1">Lein, Roland&nbsp;</span> <span>TC Rot-Gold W&uuml;rzburg&nbsp;</span></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>EU:</TD>
     <TD class="td5"> <span class="col1">Reher, Thomas&nbsp;</span> <span>TSC Werne&nbsp;</span></TD>
    </TR>
   </TABLE>
<br>
</div>
</div>
</div>
  <P><FONT size="1" face="Arial">Diese Liste wurde mit <A TARGET="_blank" HREF="http://www.TopTurnier.de">TopTurnier f&uuml;r Windows V9.6b</A> erstellt.<br></FONT>
</body>
</HTML>
