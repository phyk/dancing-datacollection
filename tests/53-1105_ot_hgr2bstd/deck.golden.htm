<!DOCTYPE html>
<html>
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
  <title>
   11.05.2024 OT, Hgr.II B Standard
  </title>
 </head>
 <body>
  <div>
   <div>
    <table>
     <tr>
      <td>
       11.05.2024 - OT, Hgr.II B Standard
      </td>
     </tr>
    </table>
   </div>
   <div>
    <div>
     Deckblatt
    </div>
    <hr/>
    <div>
     <table>
      <tr>
       <td>
        Veranstalter:
       </td>
       <td>
        Hessischer Tanzsportverband
       </td>
      </tr>
      <tr>
       <td>
        Ausrichter:
       </td>
       <td>
        Hessischer Tanzsportverband
       </td>
      </tr>
      <tr>
       <td>
        Turnierleiter:
       </td>
       <td>
        <span>
         Jungbluth, Kai
        </span>
        <span>
         Tanz-Sport-Club Fischbach
        </span>
       </td>
      </tr>
      <tr>
       <td>
        Beisitzer:
       </td>
       <td>
        <span>
         Rahaus, Markus
        </span>
        <span>
         Schwarz-Rot-Club Wetzlar
        </span>
       </td>
      </tr>
      <tr>
       <td>
        Protokoll:
       </td>
       <td>
        <span>
         tanzt, EDV-Team Hessen
        </span>
       </td>
      </tr>
      <tr>
       <td>
        Wertungsrichter:
       </td>
      </tr>
      <tr>
       <td>
        BI:
       </td>
       <td>
        <span>
         Fleischer, Georg
        </span>
        <span>
         Gr체n-Gold-Casino Wuppertal
        </span>
       </td>
      </tr>
      <tr>
       <td>
        CP:
       </td>
       <td>
        <span>
         Peinke-Dean, Lutz
        </span>
        <span>
         Tanzsportklub Residenz Dresden
        </span>
       </td>
      </tr>
      <tr>
       <td>
        DK:
       </td>
       <td>
        <span>
         Wenzel, Harald
        </span>
        <span>
         Rot-Weiss-Klub Kassel
        </span>
       </td>
      </tr>
      <tr>
       <td>
        DL:
       </td>
       <td>
        <span>
         Wied, Dr. Andrea
        </span>
        <span>
         Markgr채fler TSC, M체llheim
        </span>
       </td>
      </tr>
      <tr>
       <td>
        DR:
       </td>
       <td>
        <span>
         Zuber, Dr. Pascal
        </span>
        <span>
         TSC Metropol Hofheim
        </span>
       </td>
      </tr>
      <tr>
       <td>
        EL:
       </td>
       <td>
        <span>
         Lein, Roland
        </span>
        <span>
         TC Rot-Gold W체rzburg
        </span>
       </td>
      </tr>
      <tr>
       <td>
        EU:
       </td>
       <td>
        <span>
         Reher, Thomas
        </span>
        <span>
         TSC Werne
        </span>
       </td>
      </tr>
     </table>
     <br/>
    </div>
   </div>
  </div>
 </body>
</html>
